.github/workflows/rust-cubesql.yml

```diff
@@ -118,13 +118,12 @@ jobs:
     strategy:
       matrix:
         # Current used version + 1 LTS
-        node-version: [20, 22]
+        # TODO: Add 24 after it's been released
+        node-version: [22]
         python-version: ["3.9", "3.10", "3.11", "3.12", "fallback"]
         target: ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu"]
         # minimize number of jobs
         exclude:
-          - node-version: 20
```

Comment:
Could we comment these lines of code and write a comment "to enable it after adding 24"? Otherwise, we will forget.

Huge, thanks!